### ISSUE TITLE
Working great when built/run locally, but no events sent when installed through Google Play

### ISSUE THREAD

**Issue by skhouseholder** [danwilson/google-analytics-plugin#186]
My events show up perfectly in GA when I use `cordova run android` with my device connected via USB. But when I install the same code on the device from the Play Store, I get nothing whatsoever.

I'm only working with Android right now (obviously). My config.xml was already set to `<access origin="*" />`, but I added these for (redundant?) good measure:

    <access origin="http://*.google-analytics.com"/>
    <access origin="https://*.google-analytics.com"/>

It didn't work with either of these states. Any idea why this might be happening? Thanks!

**Comment by asantos00** [comment]
I'm having the exact same problem

**Comment by skhouseholder** [comment]
@asantos00 I never found an explanation for this, but I did get around it by completely rebuilding my project from scratch. My config.xml was a mess because I had copied it from a working phonegap (not cordova) project I'd build a year or two ago. (I didn't realize how much had changed since then and between phonegap and cordova.) Anyway, I rebuilt my project completely and re-added all the plugins one at a time (but using identical copies of the html, js, css, and a few platform-specific files). Rebuilding wasn't as much of a pain as I thought it would be.

Also possibly relevant: at the time I got this error, I was (wrongly) creating my release build with `ant release` instead of gradle. (Another side effect of me working from an old phonegap project.) If you're using cordova and ant, switching might make the difference. [Here's how I made that switch](https://github.com/floatinghotpot/cordova-admob-pro/issues/179#issuecomment-173356391).
